### PR TITLE
fix(testproviders): fix missing config for jest (@egjs/react-flicking Carousel.js)

### PR DESCRIPTION
**Pull request by mrviniciux**
## Description

Boilerplate came without some dependencies:

@testing-library/dom
and
next-auth

Not sure about @testing-library/dom but I understand that next-auth it's not something that boilerplate should add to package.json

Without these libraries I was getting errors when testing some component that was using fairplay.

image

## What was done in this PR?

- [x] Task already done
- [ ] Task to be done

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,16 +18,19 @@
...
     "@sbf/fairplay-tokens": "0.9.0",
     "@sbf/fairplay-utils": "0.4.1",
     "@sentry/nextjs": "7.7.0",
+    "@testing-library/dom": "8.11.3",
+    "next-auth": "4.9.0",
     "@testing-library/react": "12.1.2",
     "@types/next": "9.0.0",
     "axios": "0.26.0",
     "css-mediaquery": "0.1.2",
+    "identity-obj-proxy": "^3.0.0",
```
Comment (xKalleu) [inline package.json:33]: fixed version

**Review by xKalleu — APPROVED** [review]
nice 🚀

File: src/test/testUtils.tsx [hunk 2]
```diff
@@ -13,4 +44,6 @@ const customRender = (
 
 export * from '@testing-library/dom';
 
-export { act, cleanup, customRender as render };
+export { customRender as render };
+
+export { act, cleanup } from '@testing-library/react';
```
Comment (arthursakemi) [inline src/test/testUtils.tsx:49]: why are you importing `@testing-library/react` twice?

**Review by arthursakemi — APPROVED** [review]

File: src/test/testUtils.tsx (hunk 2 above)
Comment (cassiopieroni) [reply src/test/testUtils.tsx:49]: @arthursakemi here is exporting 🤔 😃

**Review by cassiopieroni — APPROVED** [review]

**Pull request merged**